src/libraries/System.Net.Quic/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
```

Comment:
What exactly does this control? We don't want this .dll to be part of our SharedFx/Targeting Pack

Reply:
Quoting @ericstj from the mail since I do not understand the process fully:
> Today this is used to bundle public API which ships in ASP.NET but builds in dotnet/runtime. This allows us to avoid putting those builds in our public packages while still giving them reference assemblies and 6.0 targeted assemblies.

Reply:
@wtgodbe if putting it into the transport package the same way as the other libraries that form the ref/runtime packs, you will likely need to exclude this one from automatically flowing in your infra.